### ISSUE TITLE
Unable to scroll up with keyboard 

### ISSUE THREAD

**Issue by siriousguy** [siriousguy/pe#11]
Since this is optimised for keyboard users and fast typers, being at the end of a really long list means that I cannot scroll up without using my mouse. Some planned enhancements can include better CLI commands such that I can scroll with the keyboard instead, benefitting more as a fast typer.

**Comment by nus-se-script** [comment]
# Team's Response
It is possible to scroll using keyboard, so this is not a feature flaw.

Steps to do this:
1. Press `tab` until one of the contacts is highlighted.
2. Use the up and down arrows to navigate up and down the contacts list.
# Items for the Tester to Verify
## :question: Issue response

Team chose [`response.Rejected`]

- [x] I disagree

**Reason for disagreement:** My consideration is that this was not mentioned in the UG or DG. For the target users of wedding planners who may not have such keyboard knowledge, they may be unable to figure that out by themselves. 

Moreover, it does not solve the problem of being able to scroll entirely up or down in one go (something like "CMD + up arrow"). Perhaps, an out-of-scope would be more valid provided that it was mentioned in the UG or DG, which in this case, was not mentioned. 

<catcher-end-of-segment><hr>
